glean-core/src/histogram/mod.rs

```diff
@@ -5,14 +5,41 @@
 //! A simple histogram implementation for exponential histograms.
 
 use std::collections::HashMap;
+use std::convert::TryFrom;
 
 use serde::{Deserialize, Serialize};
 
+use crate::error::{Error, ErrorKind};
+
 pub use exponential::PrecomputedExponential;
 pub use functional::Functional;
+pub use linear::PrecomputedLinear;
 
 mod exponential;
 mod functional;
+mod linear;
+
+/// Different kinds of histograms.
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
```

Comment:
We use to send this information in the payload, but it turned out to be unnecessary.  So, now that this is just an implementation detail, we don't *have to* do the rename since we aren't writing to a schema.  Using strings rather than ints is probably a good idea though for local data migration reasons, though.

Reply:
Oh, well, guess we can remove this then completely.